src/journal/views.py

```diff
@@ -2229,10 +2230,25 @@ def full_text_search(request):
             site=request.site_object,
         )
 
+    paginate_by = request.GET.get("paginate_by", 25)
+    if paginate_by == "all":
+        paginate_by = len(articles) if articles else 25
```

Comment:
In terms of implementation, `qs.count()` is always preferred for querysets over `len(qs)`

I wonder, however, if we really want to support "all" as a feature, since we do not have a dynamic loader for this, we could put a lot of strain on the server. I don't think the user will be happy waiting for potentially thousands of results to be returned and displayed all at once